### ISSUE TITLE
Could not find com.jetbrains.intellij.java:java-compiler-ant-tasks:211.4961.33.

### ISSUE THREAD

**Issue by NekoCaffeine** [JetBrains/gradle-intellij-plugin#592]
Hi, I am migrating my plugin to idea 2021.1 Pre, but I can't finish the build.

` build.gradle`
```groovy
buildscript {
  repositories {
    mavenCentral()
    maven { url "https://plugins.gradle.org/m2/" }
    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
    maven { url 'https://dl.bintray.com/jetbrains/intellij-plugin-service' }
    maven { url "https://jetbrains.bintray.com/intellij-third-party-dependencies" }
  }
}

plugins {
    id 'java'
    id "org.jetbrains.intellij" version "0.6.5"
}
```

`log`
```log
Execution failed for task ':instrumentCode'.
> Could not resolve all files for configuration ':detachedConfiguration3'.
   > Could not find com.jetbrains.intellij.java:java-compiler-ant-tasks:211.4961.33.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - file:/C:/Users/Administrator/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/LATEST-EAP-SNAPSHOT/f97fbeb72fc61318adbf062e2e648d57c00adf27/ideaIC-LATEST-EAP-SNAPSHOT/com.jetbrains.intellij.java/java-compiler-ant-tasks/211.4961.33/ivy-211.4961.33.xml
       - file:/C:/Users/Administrator/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/LATEST-EAP-SNAPSHOT/f97fbeb72fc61318adbf062e2e648d57c00adf27/ideaIC-LATEST-EAP-SNAPSHOT/java-compiler-ant-tasks-211.4961.33-withKotlin-withSources-withoutAnnotations.xml
       - file:/C:/Users/Administrator/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/com.jetbrains.intellij.java/java-compiler-ant-tasks-211.4961.33.xml
       - https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - https://cache-redirector.jetbrains.com/jetbrains.bintray.com/intellij-third-party-dependencies/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - https://cache-redirector.jetbrains.com/repo1.maven.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
     Required by:
         project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

**Comment by zolotov** [comment]
what is "idea 2021.1 Pre"? What exactly build you use and how?

**Comment by NekoCaffeine** [comment]
> what is "idea 2021.1 Pre"?

```groovy
intellij {
    type 'IC'
    plugins = ['java']
}
```
IntelliJ IDEA 2021.1 EAP (Community Edition)
Build #IC-211.4961.33, built on January 28, 2021

> What exactly build you use and how?

`gradle build` or `Execute the ‘build’ task in idea`

**Comment by NekoCaffeine** [comment]
`full log`
```log
4:08:39: Executing task 'build'...

> Task :compileJava UP-TO-DATE
> Task :patchPluginXml UP-TO-DATE
> Task :processResources UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :instrumentCode FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.3/userguide/command_line_interface.html#sec:command_line_warnings
4 actionable tasks: 1 executed, 3 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':instrumentCode'.
> Could not resolve all files for configuration ':detachedConfiguration3'.
   > Could not find com.jetbrains.intellij.java:java-compiler-ant-tasks:211.4961.33.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - file:/C:/Users/Administrator/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/LATEST-EAP-SNAPSHOT/f97fbeb72fc61318adbf062e2e648d57c00adf27/ideaIC-LATEST-EAP-SNAPSHOT/com.jetbrains.intellij.java/java-compiler-ant-tasks/211.4961.33/ivy-211.4961.33.xml
       - file:/C:/Users/Administrator/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/LATEST-EAP-SNAPSHOT/f97fbeb72fc61318adbf062e2e648d57c00adf27/ideaIC-LATEST-EAP-SNAPSHOT/java-compiler-ant-tasks-211.4961.33-withKotlin-withSources-withoutAnnotations.xml
       - file:/C:/Users/Administrator/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/com.jetbrains.intellij.java/java-compiler-ant-tasks-211.4961.33.xml
       - https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - https://cache-redirector.jetbrains.com/jetbrains.bintray.com/intellij-third-party-dependencies/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
       - https://cache-redirector.jetbrains.com/repo1.maven.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.4961.33/java-compiler-ant-tasks-211.4961.33.pom
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 11s
4:08:50: Task execution finished 'build'.
```

**Comment by NekoCaffeine** [comment]
Okay, I spent two minutes disabling this task better than trying to fix it.
I may be a victim of JetBrains' paid users.

**Comment by NekoCaffeine** [comment]
IntelliJ IDEA 2021.1 EAP Pre 1

I think my emphasis on Pre is tantamount to an emphasis on EAP.
You should try to reproduce the issue in the latest version instead of pointlessly obsessing over a version number description.

Thank you very much for your reply, I won't come back here to give feedback on my problem next time.

**Comment by NekoCaffeine** [comment]
Anyone who has taken a look at my posted log should be able to see "211.4961.33".
![image](https://user-images.githubusercontent.com/10548315/106332301-7b31bf00-62c1-11eb-98f7-cc738f8c5f97.png)

**Comment by zolotov** [comment]
Try to set `version` to `LATEST-EAP-SNAPSHOT`. Does it help?

```
intellij {
  verison 'LATEST-EAP-SNAPSHOT'
}
```

> Anyone who has taken a look at my posted log should be able to see "211.4961.33".

I believe the key is in __how__ you set it.

Thanks for the report by the way.

**Comment by NekoCaffeine** [comment]
> Try to set `version` to `LATEST-EAP-SNAPSHOT`. Does it help?

```log
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Where:
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Build file 'D:\Project\NekoCaffeine\Amadeus\Chocological\chocological-idea\build.gradle' line: 55
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] A problem occurred evaluating root project 'chocological-idea'.
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > No signature of method: build_78jqczz4mzwj4r8bt0vqmlw14.intellij() is applicable for argument types: (build_78jqczz4mzwj4r8bt0vqmlw14$_run_closure5) values: [build_78jqczz4mzwj4r8bt0vqmlw14$_run_closure5@5a274b4c]
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace.  Run with --scan to get full insights.
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Get more help at https://help.gradle.org
```

**Comment by zolotov** [comment]
> 2021-02-02T06:53:17.203+0800 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Build file 'D:\Project\NekoCaffeine\Amadeus\Chocological\chocological-idea\build.gradle' line: 55

Please attach `build.gradle`. I cannot interpret the output without it.

Also, note, there is a typo in my snippet of code, there is `verison` instead of `version`.

**Comment by MykytaKostiuk** [comment]
Hello @zolotov , @NekoCaffeine 
Unfortunately, we faced the same issue. 
Below you could find the stacktrace when an attempt to run "build" 
version = "LATEST-EAP-SNAPSHOT". 

```
Execution failed for task ':rt-ant:instrumentCode'.
> Could not resolve all files for configuration ':rt-ant:detachedConfiguration3'.
   > Could not find com.jetbrains.intellij.java:java-compiler-ant-tasks:211.5538.20.
     Searched in the following locations:
       - https://jcenter.bintray.com/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.5538.20/java-compiler-ant-tasks-211.5538.20.pom
       - https://repo.maven.apache.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.5538.20/java-compiler-ant-tasks-211.5538.20.pom
       - https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.5538.20/java-compiler-ant-tasks-211.5538.20.pom
       - file:/home/mykyta/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/LATEST-EAP-SNAPSHOT/368b20ac7c5f317f591f7517d78f89d25cd19b4a/ideaIC-LATEST-EAP-SNAPSHOT/com.jetbrains.intellij.java/java-compiler-ant-tasks/211.5538.20/ivy-211.5538.20.xml
       - file:/home/mykyta/.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/LATEST-EAP-SNAPSHOT/368b20ac7c5f317f591f7517d78f89d25cd19b4a/ideaIC-LATEST-EAP-SNAPSHOT/java-compiler-ant-tasks-211.5538.20-withKotlin-withSources-withoutAnnotations.xml
       - https://cache-redirector.jetbrains.com/www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.5538.20/java-compiler-ant-tasks-211.5538.20.pom
       - https://cache-redirector.jetbrains.com/jetbrains.bintray.com/intellij-third-party-dependencies/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.5538.20/java-compiler-ant-tasks-211.5538.20.pom
       - https://cache-redirector.jetbrains.com/repo1.maven.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/211.5538.20/java-compiler-ant-tasks-211.5538.20.pom
     Required by:
         project :rt-ant

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

```
plugins {
    idea
    kotlin("jvm")
    id("org.jetbrains.intellij") version "0.6.5"
}

intellij {
    type = "IU"
    version = "LATEST-EAP-SNAPSHOT"
    }
```
If you need any additional config info, please, let me know.

**Comment by NekoCaffeine** [comment]
@MykytaKostiuk 
You can finish the build by disabling the faulty task.
Here is that temporary solution, add it to `build.gradle`.
```groovy
instrumentCode.enabled = false
postInstrumentCode.enabled = false
```

**Comment by NekoCaffeine** [comment]
Incidentally, I have absolutely no use for these two stupid tasks.

**Comment by NekoCaffeine** [comment]
https://github.com/JetBrains/gradle-intellij-plugin/blob/master/src/main/groovy/org/jetbrains/intellij/tasks/IntelliJInstrumentCodeTask.groovy#L120

**Comment by zolotov** [comment]
@MykytaKostiuk thank you for the details. Indeed, as a workaround you can disable instrumentation. Do this if you don't use GUI Deisgner and ok with temporary disabling nullability assertions. In order to disable instrumentation use `intellij { instrumentCode = false }` option.

**Comment by zolotov** [comment]
@MykytaKostiuk the exception is coming from `:rt-ant` project: `Execution failed for task ':rt-ant:instrumentCode'`. But `version = "LATEST-EAP-SNAPSHOT"` is set only for root project. Is there any other place with `intellij` plugin applying in your project? If so, could you set `version = "LATEST-EAP-SNAPSHOT"` for it too? It's more correct workaround rather than disabling instrumenting tasks.

**Comment by MykytaKostiuk** [comment]
@zolotov , 
Hello, I've added 
```
intellij {
    version = "LATEST-EAP-SNAPSHOT"
}
```
to the **:rt-ant** project as well and that helped, thanks!

cc @NekoCaffeine

**Comment by zolotov** [comment]
@MykytaKostiuk thanks for the verification. the fix is coming in the next `0.7` version
